master/internal/user/external_users.go

```diff
@@ -1,12 +0,0 @@
-package user
-
-import (
-	"github.com/determined-ai/determined/master/pkg/model"
-)
-
-// UserByExternalToken returns a user session derived from an external authentication token.
-func UserByExternalToken(tokenText string,
```

Comment:
this is filled in in ee, in case you didn't see.

Reply:
will be included in my EE fix